src/terminal/terminal.c

```diff
@@ -511,6 +494,38 @@ guac_terminal* guac_terminal_create(guac_client* client,
 
 }
 
+void get_available_dimensions(guac_terminal* term, int width, int height, int* aw, int* ah, int* columns, int* rows) {
```

Comment:
It's a minor thing - but to match styling, there should be a new line between the function definition and the enclosed block.

Additionally, if possible, please wrap code at 80 characters. We generally use 8 spaces as an indent for line continuations.